### PR TITLE
refactor(cuttlefish_escript): create fetch API

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

**Comment by 6293** [comment]
maybe we should create fetch outside cuttlefish_escript , if our use case is to store conf info to ets?

if so, the arguments for `fetch` would be

```
fetch(PathToSchema, PathToConf) ->
...
```

File: src/cuttlefish_escript.erl [hunk 1]
```diff
@@ -234,20 +223,45 @@ describe_doc(Match, FindResults) ->
...
+        _ ->
             ?STDOUT("   Default Value : ~s",
-                [format_datatype(cuttlefish_mapping:default(Match),
-                    cuttlefish_mapping:datatype(Match))]);
-        false ->
-            ?STDOUT("   No default set", [])
+                [format_datatype(Default, Type)])
     end.
 
+fetch(ParsedArgs, [Query|_]) when is_list(Query) ->
```
Comment (zmstone) [inline src/cuttlefish_escript.erl:239]: I think it's better if we can push the opaque `ParsedArgs` arg to higher level.
So we can expose fetch/2 to external. e.g. `fetch(ConfFile, SchemaFile, Query)`
Then later we could also cache `load_schema` result in a ETS table for faster query.

Comment (zmstone) [reply src/cuttlefish_escript.erl:239]: and this API is better to be implemented somewhere else.
`cuttlefish_escript` is not an API module.

**Comment by 6293** [comment]
closing in favor of https://github.com/emqx/cuttlefish/pull/24

**Pull request closed without merging**